### DISCUSSION TITLE
Option to not purge generated classes?

### DISCUSSION THREAD

**Discussion by Evzin** [tailwindlabs/tailwindcss#4364] [Help]
Running build on webpack takes quite a lot of time for me (~12-15 seconds). I would prefer during development generate all styles even if they are unused for now and build with purging option enabled later.  Is such workflow possible? Somehow i can't find information on this in docs...

P.S. I know there is a css version of Tailwind available on cdn, but since i add some own classes its not the best way for me.

**Comment by simonswiss** [comment]
Hey!

This sounds like it's Taiwind's default way of working (building all classes in development, and then purging away what you didn't use for production). That's how it will behave unless you have `mode: 'jit'` enabled.

Your `build` command might set the `NODE_ENV` to `"production"`, in which case it will attempt to purge unused styles. If you don't want to rely on the value of `NODE_ENV` (default behaviour), you can [manually determine whether or not the CSS should be purged in the `purge.enabled` option property](https://tailwindcss.com/docs/optimizing-for-production#enabling-manually).

Hope it helps and answers your question!
